Documentation/README.md

```diff
@@ -4,5 +4,14 @@ SPDX-FileCopyrightText: 2024 The AMOS Projects
 SPDX-License-Identifier: MIT
 -->
 
-Build, user, and technical documentation
-Software architecture description
+# Android - Zero Code Instrumentation Observability
+
+Build, user, and technical documentation Software architecture description.
+
+## Build documentation
+
+To generate the html documentation run the following:
+
+```
+asciidoctor -r asciidoctor-diagram asciidoctor/main.adoc -o build/doc/index.html
```

Comment:
Change this to
`asciidoctor -r asciidoctor-diagram Documentation/asciidoc/main.adoc -o build/doc/index.html`
to fit the actual path of the `.adoc`.

Reply:
but from the perspective of the README.md located in Documentation the description fits. Otherwise we can put it into the main readme

Reply:
You're partly right, `asciidoctor/` has to be changed to `asciidoc/` then.

Reply:
Ah well, I am stupid...

Reply:
I moved it to main README.md and fixed the typo